### DISCUSSION TITLE
Tip needed: How to surround a selection with bracket

### DISCUSSION THREAD

**Discussion by Proful** [helix-editor/helix#5031] [Q&A] [answered]
```
G   = "goto_file_end"
```
ma" is helpful to select `"goto_file_end"`
After that selection I want to wrap with bracket `["goto_file_end"]`. How to achieve it?
```
G   = ["goto_file_end"]
```

**Comment by CptPotato** [comment, marked as answer]
Do you mean `ms[` / `ms]` to surround the selection with `[]`?

**Reply by Proful** [reply]
Thank you!
